gui/wxpython/mapdisp/properties.py

```diff
@@ -295,32 +296,38 @@ def __init__(self, parent, sbmanager):
             majorDimension=1,
             style=wx.RA_SPECIFY_COLS,
         )
-        self._setValue(self.statusbarManager.GetMode())
+        self._setValue(self._getMode())
         self._disableItems()
 
-        self.widget.Bind(wx.EVT_RADIOBOX, self._onToggleRadioBox)
+        self.widget.Bind(wx.EVT_RADIOBOX, self._onToggle)
+        self._connect()
 
     def _setValue(self, mode):
         self.widget.SetSelection(mode)
 
     def GetValue(self):
         return self.widget.GetSelection()
 
+    def _getMode(self):
+        if self.mapWindowProperty != self.statusbarManager.GetMode():
+            return self.statusbarManager.GetMode()
+        return self.mapWindowProperty
```

Comment:
Could you explain this part, what is the situation this is handling?

Reply:
It is needed for GCP group of tools, because we cannot call update when initializing the dialog, so I call only SetMode but it means that self.mapWindow Property is then different.

Reply:
I am still unsure about this, it looks unnecessarily messy... If the problem is only with GCP tools, can't you solve it there, e.g. wouldn't self.mapWindowProperties.sbItem = 5 in gcp/mapdisplay.py solve that?

Reply:
Yes, you are right, it is possible. But only when it is done after inner Map Window initialization. So, I put it as the end of the Map Display constructor.